tests/chainer_tests/functions_tests/connection_tests/test_depthwise_convolution_2d.py

```diff
@@ -79,8 +77,8 @@ def check_backward(self, x_data, W_data, b_data, y_grad):
             args = args + (b_data,)
 
         gradient_check.check_backward(
-            depthwise_convolution_2d.DepthwiseConvolution2D(
-                self.stride, self.pad),
+            lambda *x: functions.depthwise_convolution_2d(
+                *x, stride=self.stride, pad=self.pad),
```

Comment:
Use `functools.partial` or avoid the misleading name `x`.

Reply:
fixed by renaming it